src/GitHub.StartPage/StartPagePackage.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Shell;
+using GitHub.Services;
+using GitHub.UI;
+using GitHub.ViewModels;
+using System.IO;
+using Microsoft.TeamFoundation.Controls;
+using Microsoft.TeamFoundation.Git.Controls.Extensibility;
+using Microsoft.VisualStudio.Shell.CodeContainerManagement;
+using ICodeContainerProvider = Microsoft.VisualStudio.Shell.CodeContainerManagement.ICodeContainerProvider;
+using CodeContainer = Microsoft.VisualStudio.Shell.CodeContainerManagement.CodeContainer;
+using Task = System.Threading.Tasks.Task;
+using System.ComponentModel;
+using GitHub.Models;
+using GitHub.Extensions;
+using GitHub.Primitives;
+using GitHub.VisualStudio;
+
+namespace GitHub.StartPage
+{
+    [PackageRegistration(UseManagedResourcesOnly = true)]
+    [Guid(guidPackage)]
+    [ProvideCodeContainerProvider("GitHub Container", guidPackage, Guids.ImagesId, 1, "#110", "#111", typeof(GitHubContainerProvider))]
+    public sealed class StartPagePackage : ExtensionPointPackage
+    {
+        static IServiceProvider serviceProvider;
+        internal static IServiceProvider ServiceProvider { get { return serviceProvider; } }
+
+        public const string guidPackage = "3b764d23-faf7-486f-94c7-b3accc44a70e";
```

Comment:
Since the project is now in the repo, this can go into the Guids class with all the other Guids, and get a proper name like "StartPagePackageId"